src/pseudopeople/noise.py

```diff
@@ -44,32 +44,34 @@ def noise_form(
     :return:
         Noised form data
     """
-    randomness = get_randomness_stream(form, seed)
+    randomness = get_randomness_stream(form.name, seed)
 
-    noise_configuration = configuration[form.value]
+    noise_configuration = configuration[form.name]
     for noise_type in NOISE_TYPES:
-        if isinstance(noise_type, RowNoiseType):
-            # Apply row noise
-            form_data = noise_type(form_data, noise_configuration, randomness)
+        if noise_type.is_implemented:
```

Comment:
Should be unneccessary.

Reply:
Not all noise types are implemented

Reply:
True, but there are functions there. They simply return the column unnoised. This, again, is code that is unnecessary imho.